realm/realm-library/src/main/java/io/realm/ManagedMapManager.java

```diff
@@ -335,6 +335,9 @@ public Mixed put(Object key, @Nullable Mixed value) {
 
     @Override
     public boolean containsValue(@Nullable Object value) {
+        if (value == null) {
```

Comment:
Sounds like we are not able to hold `null`s. Is this documented properly somewhere?

Reply:
The way to store null values in Mixed is by using `Mixed.nullValue()`. I think it should be documented somewhere in one of the mixed classes. I will write a side note here though.